src/TurboRouter.sol

```diff
@@ -133,6 +133,10 @@ contract TurboRouter is ERC4626RouterBase, ENSReverseRecord {
         safe.sweep(to, token, amount);
     }
 
+    function sweepAll(TurboSafe safe, address to, ERC20 token) external authenticate(address(safe)) {
```

Comment:
unrelated but useful

Reply:
imo just make this the default and remove the other one

Reply:
I like having both

Reply:
cuul